docs/configure

```diff
@@ -77,8 +77,9 @@ ALIAS_DOMAINS = SELECT DISTINCT alias FROM domainalias WHERE alias = '${quote_my
 domainlist local_domains = @ : example.org : ${lookup mysql{VIRTUAL_DOMAINS}} : ${lookup mysql{ALIAS_DOMAINS}}
 domainlist relay_to_domains = ${lookup mysql{RELAY_DOMAINS}}
 hostlist   relay_from_hosts = localhost : @ : 192.168.0.0/24
-trusted_users = avleen:www
+trusted_users = vexim:www-data
```

Comment:
I wonder if we need this either. Exim docs say, that " If this option is set, any process that is running as one of the listed users is trusted", but I don't think anything is supposed to be run under the vexim user. Furthermore, [here's what trusted users can do](http://www.exim.org/exim-html-current/doc/html/spec_html/ch-the_exim_command_line.html#SECTtrustedadmin), and again, I don't think vexim needs this, do you?

On my working Debian system the following are the defaults that I never bothered to change. And everything works with them:
```
sonata:/etc/exim4/conf.d# grep -iR trusted_users
main/02_exim4-config_options:.ifndef MAIN_TRUSTED_USERS
main/02_exim4-config_options:MAIN_TRUSTED_USERS = uucp
main/02_exim4-config_options:trusted_users = MAIN_TRUSTED_USERS
sonata:/etc/exim4/conf.d#
```
I think we can safely remove this option, or at least comment it out, leaving only www-data in the commented out list. I'd actually be in favor of removal, because the smaller the config we provide, the easier it is to extend and improve for the end-user.